src/components/DropZone/DropZone.tsx

```diff
@@ -304,6 +305,9 @@ export class DropZone extends React.Component<CombinedProps, State> {
         {dragErrorOverlay}
         <div className={styles.Container}>{children}</div>
         <VisuallyHidden>
+          <label htmlFor={id}>
```

Comment:
A couple of items for the drop zone label:

- We have the `label` prop to allow consumers to add a visible label to the drop zone. This prop adds a proper `<label>` with a `for`/`id` relationship for the visually hidden input for the component. However, when the default label is added, it creates two labels for the same component in the output. Only one will get read (typically the first one in the DOM).
- How would the visually hidden default label work if there are multiple drop zone components on the same page? I think the `label` prop is there as a way to prevent duplicate labels.

Here's what the output looks like right now for the example showing a consumer-provided label:

```
<label id="DropZone12Label" for="DropZone12" class="Label-Text_SY9aZ">Theme files</label>
...
<span class="VisuallyHidden-VisuallyHidden_36MaW">
<label for="DropZone12">Upload file</label>
<input id="DropZone12" type="file" multiple="" autocomplete="off">
</span>
```

I'm thinking we could:

- Provide a default visually hidden label unless the consumer provides their own through the prop, and document recommending providing visual labels for drop zones, especially if there are multiple ones on the page
- Or make the `label` prop required, and allow consumers to make it visible or visually hidden

This is complicated by how the control had an additional visible button, and the input is off-screen when it gets focus, which is an issue (#762).

Reply:
My bad this should be a quick fix!

Reply:
I have created a new issue for making it required and fixed the issue for now when there is no label provided it uses a sensible default.